apiserver/app/createInvoice_test.go

```diff
@@ -77,3 +76,36 @@ func TestCreateInvoice(t *testing.T) {
 		t.Errorf("Response was incorrect. We expected %s", invoice)
 	}
 }
+
+func TestRateLimiter(t *testing.T) {
+	t.Cleanup(cleanDb(t))
+	a.cache.FlushAll(ctx)
```

Comment:
let's do it similarly as `cleanDb`. Create helepr function `cleanCache` nad add cleanup method